src/main/java/org/simulator/sbml/SimpleConstraintListener.java

```diff
@@ -48,6 +48,8 @@
    */
   private static final transient Logger logger = Logger.getLogger(SimpleConstraintListener.class.getName());
 
+  boolean printViolationLog = true;
```

Comment:
What happens if there are multiple constrains in a model? Is this listener assigned to exactly one constraint? Or will this switch decide if messages should be printed out for all constraints?

Also, I don't see a mechanism to switch this back on again. What happens if a constraint is, for example, violated from time 2.5 to 6.4 but subsequently not violated until time 12.3, where the condition again indicates a violation? A user would certainly be interested to see messages for each new violation period.